Tools/autotest/arduplane.py

```diff
@@ -3094,45 +3094,6 @@ def test_fence_alt_ceil_floor(self):
 
         self.fly_home_land_and_disarm(timeout=150)
 
-    def test_fence_breached_change_mode(self):
```

Comment:
Why not invert this test and ensure the mode *can* be changed after fence breach, and that it doesn't get changed back unless we breach the backup fence?

Reply:
done